freqtrade/commands/cli_options.py

```diff
@@ -182,11 +182,17 @@ def __init__(self, *args, **kwargs):
 
     ),
     "exportfilename": Arg(
-        '--export-filename',
-        help='Save backtest results to the file with this filename. '
-        'Requires `--export` to be set as well. '
-        'Example: `--export-filename=user_data/backtest_results/backtest_today.json`',
-        metavar='PATH',
+        "--export-filename",
+        help="Save backtest results to the file with this filename. "
+        "Requires `--export` to be set as well. "
+        "Example: `--export-filename=user_data/backtest_results/backtest_today.json`",
+        metavar="PATH",
+    ),
+    "backtestfilename": Arg(
+        "--backtest-filename",
```

Comment:
I'd clearly not use 2 different arguments / configuration keys for the same thing.
That's pretty dangerous, and is bound to fail assuming one comes from the configuration (then you got both variables available - and will wonder why the wrong file is loaded).

you can assign a 2nd argument to the same key eventually, and maybe change the help text to something like `Filename for backtesting results" - so it works for both loading and saving.

Reply:
you mean like this? I haven't tested commit 298815d yet, I wanted to see if I understood you right

Reply:
Yes exactly